web/app_dev.php

```diff
@@ -1,28 +1,20 @@
 <?php
 
+use Symfony\Component\Dotenv\Dotenv;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Debug\Debug;
 
-// If you don't want to setup permissions the proper way, just uncomment the following PHP line
-// read http://symfony.com/doc/current/setup.html#checking-symfony-application-configuration-and-setup
-// for more information
-//umask(0000);
-
-// This check prevents access to debug front controllers that are deployed by accident to production servers.
-// Feel free to remove this, extend it, or make something more sophisticated.
-if (isset($_SERVER['HTTP_CLIENT_IP'])
-    || isset($_SERVER['HTTP_X_FORWARDED_FOR'])
-    || ((!in_array(@$_SERVER['REMOTE_ADDR'], ['127.0.0.1', '::1']) && !substr(@$_SERVER['REMOTE_ADDR'], 0, 8) === '192.168.') || php_sapi_name() === 'cli-server')
-) {
-    header('HTTP/1.0 403 Forbidden');
-    exit('You are not allowed to access this file. Check '.basename(__FILE__).' for more information.');
-}
-
 /** @var \Composer\Autoload\ClassLoader $loader */
 $loader = require __DIR__.'/../app/autoload.php';
-Debug::enable();
 
-$kernel = new AppKernel('dev', true);
+$dotenv = new Dotenv();
+$dotenv->load(__DIR__.'/../.env');
+
+if($_ENV['APP_ENV'] !== 'prod') {
```

Comment:
why don't we just delete this file (`app_dev.php`), instead of doing weird things like that?